easybuild/easyconfigs/s/skewer/skewer-0.2.2-intel-2016b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'MakeCp'
+
+name = 'skewer'
+version = '0.2.2'
+
+homepage = 'https://github.com/relipmoc/skewer'
+description = """skewer implements the bit-masked k-difference matching algorithm dedicated to the task of adapter
```

Comment:
Yeah, they have a bold around the softwarename which makes it a bit clearer, but whatever, it doesn't really matter.